dotcom-rendering/src/components/AllEditorialNewslettersPage.tsx

```diff
@@ -44,14 +44,26 @@ export const AllEditorialNewslettersPage = ({
 				`}
 			/>
 			<SkipTo id="maincontent" label="Skip to main content" />
-			<SkipTo id="navigation" label="Skip to navigation" />
-			<Island clientOnly={true} defer={{ until: 'idle' }}>
+			<SkipTo id="navigation" label="Skip to navigation" />=
```

Comment:
guessing this is not meant to be here?

```suggestion
			<SkipTo id="navigation" label="Skip to navigation" />
```